### ISSUE TITLE
angular-cli@webpack and Materialize dialogs

### ISSUE THREAD

**Issue by pchristou** [InfomediaLtd/angular2-materialize#87]
Hi,

My setup is angular-cli with webpack and I've npm'd both Materialize CSS and angular2-materialize. Some Materialize functionality works (buttons, datepicker) but unfortunately, dialogs do not.  See image for the different variations of error. All are pointing to an issue with Velocity. I've seen issues #20 and #48 which seems to point to a jQuery conflict issue but bit lost as to how I work around this considering the angular-cli@webpack bundle is somewhat of a black box...

![dialogissues](https://cloud.githubusercontent.com/assets/18244820/17783509/4ea7597a-6570-11e6-911d-7ef2d6a3781a.PNG)

**Comment by bafinneg** [comment]
Think I could be having a similar issue however I don't get the velocity error (or anything) in the console.

Like you said, some materialize functions work - dropdowns, buttons but collapsible has problems.

- sideNav hamburger is unresponsive
- accordion dropdowns attempt to animate but immediately pull back up 

Using angular-cli@webpack1.0.0-beta.11-webpack.2
Installed materialize-css, angular2-materialize

I'm not sure if angular2-materialize is installed correctly as some files have changed since the angular-cli webpack update.

differences from the install steps:

Installing & configuring angular2-materialize in projects created with angular-cli
- angular-cli-build.js has been removed 
- system-config.ts has been removed 

Installing and configuring angular2-materialize with webpack
- webpack.config.js does not exist in angular-cli webpack install

**Comment by pchristou** [comment]
No need to apologise @bafinneg we're all after the same thing - a solution! :-)

I managed to get my dialogs working. Rather than trying to cobble together a fix, I simply installed a fresh copy of the original angular2 seed project and then installed materialize separately as noted here: https://github.com/InfomediaLtd/angular2-materialize 

After that, I applied the "fix" supplied by @camarrone here: https://github.com/InfomediaLtd/angular2-materialize/issues/20 copied over my ts files and boom, all was well. I haven't tried sidenavs since I don't need them but everything else works so far... Hope this helps!
